src/include/migraphx/op/name.hpp

```diff
@@ -37,7 +37,11 @@ struct op_name
 {
     std::string name() const
     {
+#if MIGRAPHX_ASAN
+        std::string name = get_type_name<Derived>();
+#else
         static const std::string& name = get_type_name<Derived>();
+#endif
```

Comment:
I dont think this should be here. I think there was a mistake with the merge.

Reply:
Removed